src/forms-opportunities/oppty-handlers/low-conversion-handler.js

```diff
@@ -162,24 +166,8 @@ export default async function createLowConversionOpportunities(auditUrl, auditDa
         log.debug('Forms Opportunity high form views low conversion updated');
       }
 
-      log.info('sending message to mystique');
-      const mystiqueMessage = {
-        type: 'guidance:high-form-views-low-conversions',
-        siteId: auditData.siteId,
-        auditId: auditData.auditId,
-        deliveryType: site.getDeliveryType(),
-        time: new Date().toISOString(),
-        data: {
-          url: opportunityData.data.form,
-          cr: opportunityData.data.trackedFormKPIValue,
-          metrics: opportunityData.data.metrics,
-          form_source: opportunityData.data.formsource || '',
-        },
-      };
-
       // eslint-disable-next-line no-await-in-loop
-      await sqs.sendMessage(env.QUEUE_SPACECAT_TO_MYSTIQUE, mystiqueMessage);
-      log.info(`forms opportunity high form views low conversions sent to mystique: ${JSON.stringify(mystiqueMessage)}`);
+      await sendMessageToFormsQualityAgent(auditDataObject, context, opportunityData);
```

Comment:
We should call this method from a central location after we call create<oppoty-typ> for all the oppty